controller-server/src/test/java/com/yahoo/vespa/hosted/controller/maintenance/UpgraderTest.java

```diff
@@ -504,6 +504,9 @@ public void testVersionIsBrokenAfterAZoneIsLive() {
         tester.readyJobTrigger().maintain();
         tester.readyJobTrigger().maintain();
         tester.readyJobTrigger().maintain();
+        tester.readyJobTrigger().maintain();
+        tester.readyJobTrigger().maintain();
+        tester.readyJobTrigger().maintain();
```

Comment:
Can we add the `tester.triggerAllJobs` method soon please? 🙂

Reply:
Yes ...